less/index.less

```diff
@@ -54,6 +54,65 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	
 }
 
+// ======== === === == =====Variables
+@width: 10px;
+@height: @width + 10px;
+
+
+// header {
+// 	width:@width;
+// 	height: @height;
+// }
+// variables
+
+@color:lime;
+// @bodyFontStack: font-family:"Supria Sans", Helvetica, Arial, sans-serif;
+
+header {
+	nav {
+		.centerflex;
+		.border;
+	}
+}
+
+body {
+	@media (max-width:500px) {
+		width:100%;
+		background-color: aliceblue;
+	}
+	background-color: aquamarine;
+	font-size: 33px;
+	border: 1px solid black;
+	padding: 20px;
+	.work-history {
+		color:red;
+		font-size: 30px;
+		margin: 20px;
+		ul{
+			width: 400px;
+			height: 100px;
+			background:@color;
+			
+		}
+	} //end body selector
+
+}
+// mixins
```

Comment:
I highly recommend putting your mixin below or above your variables, it's better to practice to put such things at the top of your code and now randomly in the middle like it is here. I would move this code above line 74.